README.md

```diff
@@ -167,6 +167,33 @@ app.sessionEnded(function(request, response) {
 });
 ```
 
+## AudioPlayer Event Request
+
+Define the handler for multiple events using multiple calls to `audioPlayer()`. You can define only one handler per event. Event handlers that don't return an immediate response (because they do some asynchronous operation) must return false. 
```

Comment:
Hey @fremail , thank you for fixing my sloppy code and adding the AudioPlayer Event Request feature! 

I would love to use the feature, however having issues issues with my app listening to AudioPlayer events.  When the Alexa sends a audioplayer request (I can log the request_json, so I know its sent), the try / catch block with the `var requestType = request.type()` isn't even firing in my code.

Am I missing something here?